src/components/radio-button/styles.js

```diff
@@ -15,10 +15,12 @@ export const sizeClassNames = {
 	sm: {
 		checkbox: 'size-4',
 		icon: 'size-1.5',
+		info: 'w-4 h-4',
```

Comment:
```suggestion
		info: 'size-4',
```

Reply:
@BabuTechBsf, This is just a suggestion. You can skip it if you want. Same for the other size class name suggestion.